dynawo/sources/Solvers/Common/test/TestBasics.cpp

```diff
@@ -79,6 +79,8 @@ TEST(SimulationCommonTest, testSolverCommon) {
   smj.addTerm(2, 3.);
   smj.changeCol();
   smj.addTerm(1, 4.);
+  SparseMatrix::CheckError check_status = smj.check();
```

Comment:
I would have expected a unit test with a home-made submodel?

Reply:
I just integrated the new API in the already existing unit test. Since this unit test doesn't check the fucntionnality of retrieving the info of the equation, there is no need to use a custom submodel.

Reply:
then please add this missing test